### ISSUE TITLE
Bug in order number display at confirmation !

### ISSUE THREAD

**Issue by myriamboure** [openfoodfoundation/openfoodnetwork#1633]
With the latest release there is a bug when I make an order, I see on the screen the order confirmation but the order number doesn't show, a strange text display instead.
![screenshot from 2017-06-21 09-53-39](https://user-images.githubusercontent.com/8135376/27373236-d4e6a020-5667-11e7-80e4-f3942b74c1d5.png)
On the email I receive as a customer, this is o I see the order number, but it's confusing and looks like a bug.
@sstead do you have the same issue in Aus ? Just to know if that's a general bug or if it's related to France..

**Comment by myriamboure** [comment]
Any idea where that might come from @ltrls or @pacodelaluna or @clairezed ?

**Comment by myriamboure** [comment]
we have only upgraded staging in France, waiting to solve that before we push to production

**Comment by daniellemoorhead** [comment]
Sally is away this week @myriamboure but I've asked @mkllnk to have a look at this.

**Comment by mkllnk** [comment]
@myriamboure It seems to be a missing translation. I just checked the transifex branch and it didn't receive any updates. I checked the translation status of French and it says it is at 94%. Very strange is that I looked a few minutes ago and thought it reported 100%. I can't verify that now.

The way forward is to translate to 100%. That should trigger a new commit on the transifex branch which can then be deployed and should solve your problem.
